pages/common/codecrafters.md

```diff
@@ -0,0 +1,16 @@
+# codecrafters
+
+> Practice writing complex software.
+> More information: <https://codecrafters.io/>.
```

Comment:
```suggestion
> More information: <https://docs.codecrafters.io/cli/usage>.
```

Reply:
Our cli is pretty simple. If people go to the doc page, they will get the same info as this tldr page, so not much value in pointing to doc.